scripts/eosio_build_fedora.sh

```diff
@@ -158,7 +158,7 @@
 		cd ..
 		mkdir build
 		cd build
-		cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=${HOME}/opt/wasm -DLLVM_TARGETS_TO_BUILD= -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly -DCMAKE_BUILD_TYPE=Release ../
+		cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=${HOME}/opt/wasm -DLLVM_ENABLE_RTTI=1 -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly -DCMAKE_BUILD_TYPE=Release ../
```

Comment:
This seems to imply that we are using this LLVM install for 2 different purposes which we try not to.  Either it is being used to build something or it is being linked against for our wasm-jit library.  Lets figure out if there is a solution that has better isolation like our other platforms (hopefully) do.